doc/iterations/2024.1/shapes/add_support_for_libraries.adoc

```diff
@@ -0,0 +1,39 @@
+= Add the support for libraries
+
+== Context
+
+Users want to have access to read only data (a.k.a libraries).
```

Comment:
I would say store and retrieve read only data because we will need to discuss about the storing part too

Reply:
done